packetbeat/protos/dns/dns.go

```diff
@@ -41,6 +42,11 @@ const (
 // Transport protocol.
 type Transport uint8
 
+var (
+	unmatchedRequests  = expvar.NewInt("dnsUnmatchedRequests")
+	unmatchedResponses = expvar.NewInt("dnsUnmatchedResponses")
```

Comment:
general idea: should we use the same naming rules for expars as we do metricbeat?